server/historian/packages/historian-base/src/runnerFactory.ts

```diff
@@ -48,9 +49,21 @@ export class HistorianResourcesFactory implements utils.IResourcesFactory<Histor
         const riddlerEndpoint = config.get("riddler");
         const riddler = new historianServices.RiddlerService(riddlerEndpoint, tenantCache);
 
+        const throttleMaxRequestsPerMs = config.get("throttling:maxRequestsPerMs") as number || undefined;
```

Comment:
This should be `as number | undefined` because it is a type declaration of the `config.get()` return value, which is `any` and can be `undefined`. The result of the code here could change `0` to `undefined`, but should instead be  telling the TypeScript compiler that this value might be `undefined`.